src/main/java/core/basesyntax/models/Fruit.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.models;
+
+public record Fruit(String fruitName) implements Comparable<Fruit> {
+
+    @Override
+    public int compareTo(Fruit fruit) {
+        return this.fruitName.compareTo(fruit.fruitName);
+    }
```

Comment:
I do not think we need all of that, this class has only 1 string field - so it can easily be replaced with just string in storage, or if you want to make use of it, add `amount` or `quantity` field here, and make `storage` not a `map`, but a `list of fruits`.
But I would prefer the first approach